cmd/kueuectl/app/list/helpers.go

```diff
@@ -49,3 +58,30 @@ func addActiveFilterFlagVar(cmd *cobra.Command, p *[]bool) {
 	cmd.Flags().BoolSliceVar(p, "active", make([]bool, 0),
 		"Filter by active status. Valid values: 'true' and 'false'.")
 }
+
+func addForObjectFlagVar(cmd *cobra.Command, p *string) {
+	cmd.Flags().StringVar(p, "for", "",
+		"Filter events to only those pertaining to the specified resource.")
+}
+
+func parseForObjectFlag(input string) (objectRef, error) {
+	var objRef objectRef
+	if input == "" {
+		return objRef, nil
+	}
+
+	parts := strings.Split(input, "/")
```

Comment:
Isn't there a parser for this from the kubectl library?

Reply:
I saw the [schema](https://pkg.go.dev/k8s.io/apimachinery@v0.29.5/pkg/runtime/schema) package is being used in some parts but couldn't find a parser that matches with the format we're using `[TYPE[.API-GROUP]/]NAME`.